python/cudf/cudf/pandas/fast_slow_proxy.py

```diff
@@ -681,6 +681,24 @@ def __matmul__(self, other):
     def __rmatmul__(self, other):
         return _fast_slow_function_call(operator.matmul, other, self)[0]
 
+    def __isub__(self, other):
```

Comment:
Why are only `__isub__` and `__iadd__` covered in this PR? It seems that other in-place operators exist.

```python
>>> import pandas as pd
>>> s = pd.Series([1, 2, 3])
>>> s *= 3  # __imul__
>>> s
0    3
1    6
2    9
dtype: int64
>>> s **= 2  # __ipow__
>>> s
0     9
1    36
2    81
dtype: int64
>>> [x for x in dir(s) if x.startswith("__i")]
['__iadd__', '__iand__', '__ifloordiv__', '__imod__', '__imul__', '__init__', '__init_subclass__', '__int__', '__invert__', '__ior__', '__ipow__', '__isub__', '__iter__', '__itruediv__', '__ixor__']
```

Reply:
Thanks for catching this Bradley!!! I added the missing ones.